app/src/main/java/org/mozilla/fenix/settings/SyncPreference.kt

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import androidx.appcompat.widget.SwitchCompat
+import androidx.preference.PreferenceViewHolder
+import androidx.preference.SwitchPreferenceCompat
+import org.mozilla.fenix.R
+
+/**
+ * Variation of [SwitchPreferenceCompat] that uses a custom widgetLayoutResource in order to implement
+ * visibility changes to it.
+ * */
+class SyncPreference @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null
+) : SwitchPreferenceCompat(context, attrs) {
+
+    private var switchView: SwitchCompat? = null
+
+    /**
+     * Whether or not switch's toggle widget is visible.
+     * */
```

Comment:
``
```suggestion
     */
```

Reply:
![image](https://user-images.githubusercontent.com/60002907/117108098-5bf87100-ad8b-11eb-9b4c-c990dc686f91.png)
Most of them mine. I'm not sure why this is happening as the kdoc Editor setting is correct:
![image](https://user-images.githubusercontent.com/60002907/117108481-e8a32f00-ad8b-11eb-9c09-aad80f9abd44.png)
I suggest we fix this in a follow-up issue.

Reply:
Can you file this instead? Sorry!